examples/animation/map.featureanimation.ts

```diff
@@ -111,11 +111,9 @@ function addFeatureAt(p: Coordinate) {
       break;
   }
 
- //TODO: Indexing does not yet work
- /*
```

Comment:
@smellman (CC: @Siedlerchr)  
This featureanimation example seems to be still on the way in #72, and I encountered the following error when accessing http://localhost:8181/examples/animation/map.featureanimation.html on my local environment, so I think that it's better to keep (recover) this block comment (`/* ~ */`).
<img width="1155" alt="featureanimation-example-error" src="https://user-images.githubusercontent.com/629923/163700060-58768116-a093-4451-b016-669ad91eb41e.png">

Reply:
No, I think so because I fixed this problem in [here](https://github.com/Siedlerchr/types-ol-ext/pull/84/commits/5479b485c1741c68aa848725bc55cceb345b8176).

Reply:
Okay, thanks for the comment.
Since this PR is merged already, so I resolve this conversation. 🙇‍♂️